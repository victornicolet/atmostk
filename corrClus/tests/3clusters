1 2 3 4 5 6 7 8
1 2 3
6 8 -3
1 7 2
7 6 -1
5 6 1
4 3 1
2 8 -3
7 5 -1
