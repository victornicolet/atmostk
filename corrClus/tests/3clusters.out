1 2 7
3 4 8
6 5